volcli/volcli.go

```diff
@@ -125,7 +135,7 @@ func TenantList(ctx *cli.Context) {
 // VolumeCreate creates a new volume with a JSON specification to store its
 // information.
 func VolumeCreate(ctx *cli.Context) {
-	if len(ctx.Args()) != 2 {
+	if len(ctx.Args()) != 1 {
 		errExit(ctx, fmt.Errorf("Invalid arguments"), true)
```

Comment:
I understand that this comment is unrelated to the PR but thought will share since I noticed it :) but feel free to track it through separate issue/PR. 

This error might be more useful if it states something like `volcli expects only one argument` ? Same is true at multiple places below. May be we could use a variable/constant here.